examples/device/ep/csrc/vmm.cpp

```
@@ -25,22 +25,130 @@ namespace {
+    size_ = nixl_ep::align_up<size_t>(size, ctx.granularity);
+
+    CUresult st = cuMemCreate(&handle_, size_, &ctx.prop, 0);
+    if (st != CUDA_SUCCESS) {
+        log_cu_api(st, cu_api_log_level::warning, k_vmm_ctx, "failed to create CUDA VMM allocation");
+        return false;
+    }
+
+    st = cuMemAddressReserve(&ptr_, size_, 0, 0, 0);
+    if (st != CUDA_SUCCESS) {
+        log_cu_api(st, cu_api_log_level::warning, k_vmm_ctx, "failed to reserve CUDA virtual address");
+        release();
```

Cppcheck (2.20.0)

[style] The function 'clean_meta' is never used.

(unusedFunction)